docs/source/guides/configuration.rst

```diff
@@ -83,6 +83,13 @@ Environment-specific parameters
   track a transaction if for some reason it is not updated with new rounds.
   However large values increase the average number of connected clients during
   each round.
+- ``"initial_peers`` is an optional parameter specifying list of peers a node will
+  use after startup instead of peers from genesis block. It could be useful when
+  you add a new node to the network where the most of initial peers became
+  malicious. Peers list should be provided as a JSON array:
```

Comment:
I would prefer "may become malicious" as if we generally do not know who exactly is going to cheat.